src/controllers/midi/midicontroller.cpp

```diff
@@ -199,6 +200,16 @@ void MidiController::commitTemporaryInputMappings() {
 
 void MidiController::receive(unsigned char status, unsigned char control,
                              unsigned char value, mixxx::Duration timestamp) {
+    if (userActivityInhibitTimer.isNull()) {
+        mixxx::ScreenSaverHelper::triggerUserActivity();
+        userActivityInhibitTimer.start();
+    }
+     // Inhibit Updates for 1000 milliseconds
+    if (userActivityInhibitTimer.elapsed() > 1000) {
+        mixxx::ScreenSaverHelper::triggerUserActivity();
+        userActivityInhibitTimer.start();
+    }        
```

Comment:
MidiController inherits from Controller. So this code should be moved to a Controller function and called from here.